macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -27,6 +27,21 @@
                                                                                                    run_result_dict,
                                                                                                    flatten_test_node)) %}
                     {%- endif -%}
+                {% elif flatten_test_node.short_name == 'dimension_anomalies' %}
```

Comment:
Is there a reason that this is a new 'if' and not just adding 'dimension_anomalies' to the list on row 14?

Reply:
There is a different method we use just for the dimension, but i moved it inside the if in line 14 🙂